platformtour/platformtour.py

```diff
@@ -0,0 +1,170 @@
+"""
+This is the core logic for the Platform Tour xblock, which introduces students to
+a course through a digital tour.
+"""
+import json
+import pkg_resources
+from django.template import Context
+from django.template.loader import get_template
+from xblock.core import XBlock
+from xblock.fields import List, Scope, String
+from xblock.fragment import Fragment
+
+import default_steps
+
+def _resource_string(path):
+    """Handy helper for getting resources from our kit."""
+    data = pkg_resources.resource_string(__name__, path)
+    return data.decode("utf8")
+
+
+class PlatformTourXBlock(XBlock):
+    """
+    Allows students to tour through the course and get familiar with the
+    platform.
+    """
+
+    display_name = String(
+        display_name=('Display Name'),
+        help=(
+            'This is the title for this component'
+        ),
+        default='Platform Tour',
+        scope=Scope.settings,
+    )
+
+    button_label = String(
+        display_name=('Button label'),
+        help=('This is the text that will appear on the button'),
+        default='Begin Platform Tour',
+        scope=Scope.settings,
+    )
+
+    intro = String(
+        display_name=('Introduction text'),
+        help=('This is the introduction that will precede the button'
+            ' and explain its presence to the user'),
+        default='Click the button below to learn how to navigate the platform!',
+        scope=Scope.settings,
+    )
+
+    step_choices = List(
```

Comment:
`enabled_default_steps` or `chosen_default_steps` maybe, to be a bit more explicit?

Reply:
how about `chosen_steps`?